source/installguide/management-server/_database.rst

```diff
@@ -287,6 +301,12 @@ same node for MySQL. See `“Install the Database on the Management Server Node
 
       service mysqld start
 
+   On SUSE, start MySQL
+
+   .. parsed-literal::
+
+      systemctl start mysql
```

Comment:
Do you also need "systemctl enable mysql" - is the start of MySQL enabled on boot (check with systemctl status MySQL - it is reported there "vendor" boot on start behaviour and the current/set behaviour on boot - if it's set to start automatically or not)

Reply:
Added the line to enable it as well